app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationRouter.kt

```diff
@@ -101,16 +96,61 @@ fun NewConversationRouter(newConversationViewModel: NewConversationViewModel = h
                                         contact = searchContact.contact,
                                         internal = searchContact.internal
                                     )
-                                }
+                                },
+                                onNewGroupClicked = { newConversationViewModel.openNewGroupScreen() }
                             )
+
                         }
                     )
                 }
-            }
-        )
+            )
+            composable(
+                route = NewConversationStateScreen.NEW_GROUP,
+                content = {
+                    NewGroupScreen(onBackPressed = { newConversationState.navigateBack() })
+                }
+            )
+
+        }
     }
 }
 
+@Composable
+fun screenWithSearchTopBar(
```

Comment:
Let's upper case composable components per convention
```suggestion
fun ScreenWithSearchTopBar(
```